### ISSUE TITLE
[bug/web-app] tagging @linuxxx and @linux results in tagging @linux twice

### ISSUE THREAD

**Issue by 7twin** [devRant/devRant#431]
As the title says, if you put a tag to both linuxxx and linux, it tags linux twice:

![image](https://user-images.githubusercontent.com/32747235/40211951-c8d4f878-5a4d-11e8-9786-3519416559f5.png)

**Comment by Skayo** [comment]
Not in the app

**Comment by 7twin** [comment]
@Skayo yes, only the webapp seems to be affected, so I am glad I wasn't wrong about mentioning it in the title

**Comment by Skayo** [comment]
@7twin Sorry, didn't see it.

**Comment by 7twin** [comment]
@Skayo wasn't meant to be sarcastic, don't worry
